instagram/src/components/PostsContainer/PostContainer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Post from "./Post.js";
+
+const PostsContainer = props => {
+  return (
+    <div className="posts-container-wrapper">
+      {props.posts.map((post, i) => {
+        return (
+          <div>
+            <div>{post.username}</div>
+            <img src={post.imageUrl} alt="post"/>
+            <Post key={i} post={post} />{" "}
```

Comment:
I don't see a Post.js file anywhere in this repo, but you're referencing it here and on line 2.